tsconfig.json

```diff
@@ -15,5 +15,5 @@
     "allowSyntheticDefaultImports": true,
     "baseUrl": "."
   },
-  "include": ["./src/**/*", "types.d.ts", "storybook/stories"]
+  "include": ["./src/**/*", "types.d.ts"]
```

Comment:
don't include `storybook`

we want types folder to be flat rather than
- src/index
- storybook/*